webapp/services/add_user.py

```diff
@@ -29,17 +29,16 @@ def post(self):
 				name = user_name, 
 				payment_key = user_payment_key,
 				email = user_email,
+				#date_of_birth = datetime.strptime(user_date_of_birth, '%m/%d/%Y')
 				date_of_birth = datetime.strptime(user_date_of_birth, '%Y-%m-%d %H:%M:%S')
```

Comment:
Actually it was used here. See here I'm commenting it out. So perhaps after deploying we can delete the the commented out format along with the retrofit configuration stuff.